google_cloud_http_target_task.js

```diff
@@ -0,0 +1,63 @@
+const {CloudTasksClient} = require('@google-cloud/tasks');
```

Comment:
this probably shouldn't live at the root level, maybe instead at `.github/workflow-scripts`?

Reply:
Done. I added it inside .github/workflow